packages/fxa-settings/src/constants/index.tsx

```diff
@@ -3,16 +3,8 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 export const ROOTPATH = '/';
```

Comment:
Definitely not a requirement but could go ahead and change this to `ROOT_PATH` to match, it's only used in one place. (... man, we could have used that more with the React conversion work just for consistency, heh)

Reply:
Eh, if it's used in just one spot, maybe we just remove the constant?